bfd/src/test/java/gov/cms/ab2d/bfd/client/BlueButtonClientTestSTU3.java

```diff
@@ -110,72 +101,64 @@ public static void setupBFDClient() throws IOException {
         );
 
         for (String patientId : TEST_PATIENT_IDS) {
-            createMockServerExpectation(
+            MockUtils.createMockServerExpectation(
                     "/v1/fhir/Patient/" + patientId,
                     HttpStatus.SC_OK,
-                    getRawXML(SAMPLE_PATIENT_PATH_PREFIX + patientId + ".xml"),
+                    getRawJson(SAMPLE_PATIENT_PATH_PREFIX + patientId + ".json"),
```

Comment:
A little confused about why this works when the xml parser is specified above.

Reply:
Because it doesn't use the above one, it uses the one created in the BFD search client. That's why I changed it. I would have had to support both JSON and XML in the codebase for this to work and I didn't see the reason to add a bunch of unneeded functionality just so the test would run.